### ISSUE TITLE
add visitors to user model

### ISSUE THREAD

**Issue by ThomasWaldmann** [ThomasWaldmann/pretalx#2]
currently pretalx only knows organizers and speakers, add visitors.

**Comment by ThomasWaldmann** [comment]
Ideas:

- a visitor is a speaker who did not (yet) submit a suggestion for a talk. if we think of ad-hoc talks (like lightning-talks, unconference talks, etc.) this models reality in a good way.

- a speaker is also a visitor, of course (we do not assume any minimum amount of "visiting activity", so why not)

So, we use the same model, just extend as needed.

**Comment by ThomasWaldmann** [comment]
The visitor
- is a potential speaker
- we also need the visitor-event coupling
- has a bio, too (which might mention different things depending on the event)

So, guess we also use the `SpeakerProfile` model (and rather think of it as "UserProfile" in general).

**Comment by ThomasWaldmann** [comment]
Please check my analysis.

In case we do not find a flaw, this ticket boils down to changing a few comments / docstrings that make the above stuff clear inside the source code.

**Comment by twgaras** [comment]
Agreed but am I seeing it right that every SpeakerProfile is Event specific via a ForeignKey link?
If so we should rework that as well.

**Comment by twgaras** [comment]
Wait never mind, a Visiter is a User to a specific Event

**Comment by ThomasWaldmann** [comment]
Yeah, the profile **needs** to be event-specific.

**Comment by ThomasWaldmann** [comment]
OK, considering this **DONE**.

Leaving the issue open for now in case there are questions / new requirements for User / Profile.
